config/samples/core_v1alpha1_humiofeatureflag.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: core.humio.com/v1alpha1
+kind: HumioFeatureFlag
+metadata:
+  name: humiofeatureflag-example
+spec:
+  managedClusterName: example-humiocluster
+#  externalClusterName: example-humiocluster
```

Comment:
Is this intentional?

Reply:
Yes, it's there to exemplify that one may use both